File: Python For Everyone, Horstmann/Chapter2/P2.2.py
# Write a program that computes and displays the perimeter of a letter­size (8.5 × 11
# inch) sheet of paper and the length of its diagonal.

from math import sqrt

paper_width_inches = 8.5
paper_length_inches = 11.0

circumference = (2 * paper_width_inches) + (2 * paper_length_inches)

diagonal = sqrt((paper_width_inches * paper_width_inches) + (paper_length_inches * paper_length_inches))

print("The circumference is %.4f inches " % circumference)
print("The diagonal is %.4f inches" % diagonal)

File: Java-for-Everyone-Late-Objects-Horstmann/Chapter-2-Fundamental-Data-Types/P2_2.java
/*Write a program that computes and displays the perimeter of a letter­size (8.5 × 11 inch) 
sheet of paper and the length of its diagonal.*/


public class P2_2 {

	public static void main(String[] args) {
		final double PAPER_WIDTH_INCHES = 8.5;
		final double PAPER_LENGTH_INCHES = 11.0;
		
		double perimeter = 2 * PAPER_LENGTH_INCHES + 2 * PAPER_WIDTH_INCHES;
		double diagonal = Math.sqrt(Math.pow(PAPER_WIDTH_INCHES, 2) + Math.pow(PAPER_LENGTH_INCHES, 2));
		System.out.printf("Perimeter: %.2f; Diagonal length: %.2f", perimeter, diagonal);
	}

}
